files/main.sh

```diff
@@ -44,18 +53,4 @@ curl "<%= script.url %>" | /bin/bash
   <% end %>
 <% end %>
 
-mkdir -p /tmp/configs
```

Comment:
I don't have anything making use of it, that makes sense though. Might be worth leaving in for examples sake though? Don't see it being a huge issue being taken out!